src/components/UI/Inputs/CheckBoxInput/index.tsx

```diff
@@ -83,14 +83,18 @@ const CheckBoxInput = React.forwardRef<HTMLInputElement, ICheckBoxInputProps>(
     },
     ref
   ) => {
-    const patchedContentProps = Object.assign({}, contentProps, {
-      border: false,
-      pad: {
-        horizontal: 'none',
-        vertical: 'xsmall',
+    const patchedContentProps = Object.assign(
+      {},
+      {
+        border: false,
+        pad: {
+          horizontal: 'none',
+          vertical: 'xsmall',
+        },
+        background: 'none',
       },
-      background: 'none',
-    });
+      contentProps
+    );
```

Comment:
Changed the order to allow users to customize the padding. (I also did the same for the `RadioInput` component, since it works in the same way)